Source/OpenTK/Platform/SDL2/Sdl2Factory.cs

```diff
@@ -34,7 +34,8 @@ namespace OpenTK.Platform.SDL2
 {
     class Sdl2Factory : PlatformFactoryBase
     {
-        readonly Sdl2InputDriver InputDriver = new Sdl2InputDriver();
+        private readonly object inputDriverLock = new object();
+        private Sdl2InputDriver inputDriver = new Sdl2InputDriver();
```

Comment:
Yes definitely, good catch!
Sorry, must have confused myself while doing the line-by-line commit picking...